Documentation/Signoffs/OEM Component- Auto_Light-signoff.md

```diff
@@ -0,0 +1,55 @@
+OEM Component Subsystem- Auto Light
+------
+
+## The function of the Auto Light Subsystem
+The Master microcontroller will receive the output of the Nissan Leaf AutoLight in this part of the subsystem. Furthermore, the Master microcontroller will detect if the headlight is off, on, blinking, and high and low beams.
+
+### Specification & constraints
+   - Must detect high beam,low beam, left indicator, and right indicator
+   - Must use OEM componets as the inputs
+   - The signal input must not exceed 5V 
+   
+# Auto Light Wiring schematic
+![opel2437](https://user-images.githubusercontent.com/101354063/205468261-7fca3396-009b-4eb8-9960-962065d22ac8.png)
+### Figure 1: Headlamp(High and low Relay)
+
+# Design: A Voltage divider from Auto Light to the Microcontroller
+![Capture](https://user-images.githubusercontent.com/101354063/205472902-57cf6121-181d-4fe0-904c-63295cc9c374.PNG)
```

Comment:
Nice looking schematic! And the design is good. However, consider what happens if a voltage spike were to come through. Look into opto-isolation. That way, if a voltage spike came through the headlamp bus, your system would not be damaged. 

So, you could add voltage variation between x and y volts in the headlamp supply must not damage the monitoring system as a constraint.